client/my-sites/stats/activity-log/index.jsx

```diff
@@ -277,6 +280,20 @@ class ActivityLog extends Component {
 	};
 
 	/**
+	 * The link to download a backup in ellipsis menu in ActivityLogDay > ActivityLogItem uses
+	 * this method to point that a backup has been requested and show the dialog.
+	 *
+	 * @param {string} activityId Id of the activity up to the one we're downloading.
+	 * @param {object} from       Context for tracking.
+	 */
+	handleRequestBackup = ( activityId, from ) => {
+		const { recordTracksEvent, requestBackup, siteId } = this.props;
+
+		recordTracksEvent( 'calypso_activitylog_backup_request', { from } );
+		requestBackup( siteId, activityId );
```

Comment:
is there a need to include the tracking code here or could we pass it in by wrapping `requestBackup()` in `connect()`? what is the `from` like here, do we have example data?